fetch-in-react-practice/client/src/components/Todos.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import React, { useEffect, useState } from 'react';
+import PageTitle from './PageTitle';
+import TodoList from './TodoList';
+import TodoForm from './TodoForm';
+
+export default function Todos() {
+  const [todos, setTodos] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+
+  /* Implement useEffect to fetch all todos. Hints are at the bottom of the file. */
+  useEffect(() => {
+    async function fetchTodos() {
+      try {
+        const response = await fetch('/api/todos');
+        const result = await response.json();
+        setTodos(result);
+      } catch (error) {
+        setError(error);
+      } finally {
+        setIsLoading(false);
+      }
+    }
+    fetchTodos();
+  }, []);
+
+  /* Implement addTodo to add a new todo. Hints are at the bottom of the file. */
+  async function addTodo(newTodo) {
+    try {
+      const response = await fetch('/api/todos', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(newTodo),
+      });
+      const result = await response.json();
+      const array = todos.concat(newTodo);
+      setTodos(array);
+      console.log(newTodo);
+    } catch (error) {
+      setError(error);
+    } finally {
+      setIsLoading(false);
```

Comment:
This isn't relevant here since isLoading will always be false when this function runs. (The UI won't display the Add button if isLoading is true.)